GeeksCoreLibrary/Modules/GclReplacements/Services/StringReplacementsService.cs

```diff
@@ -366,6 +370,10 @@ public string DoReplacements(string input, IDictionary<string, object> replaceDa
                 var value = Convert.ToString(replaceData[variableName], new CultureInfo("en-US"));
                 if (String.IsNullOrWhiteSpace(value))
                 {
+                    if (!variable.DefaultValue.IsNullOrEmpty())
```

Comment:
Graag hier gewoon `String.IsNullOrEmpty(variable.DefaultValue)` gebruiken. Deze extension method zorgt voor een extra `using` bovenin voor één functie (`using Microsoft.IdentityModel.Tokens`), en bovendien zorgt dit voor inconsistentie omdat we dit nergens anders gebruiken.